src/interface/swarm.cpp

```diff
@@ -135,7 +202,18 @@ void Swarm::Remove(const std::string &label) {
   }
 }
 
+template <typename T>
+void Swarm::ResizeParArray(ParArrayND<T> &var, int n_old, int n_new) {
+  auto oldvar = var;
+  auto newvar = ParArrayND<T>(oldvar.label(), n_new);
+  GetBlockPointer()->par_for(
+      "ResizeParArray", 0, n_old - 1,
+      KOKKOS_LAMBDA(const int n) { newvar(n) = oldvar(n); });
+  var = newvar;
+}
```

Comment:
From the documentation of `Kokkos::resize()`: `...will not preserve content.`, which is unfortunate because that's what we want. I don't see a similar content-preserving resize in Kokkos. Maybe this functionality should be moved up from here to `ParArrayND` at some point?

Reply:
You're using this now, right?